x-pack/plugins/security_solution/public/common/containers/matrix_histogram/index.ts

```diff
@@ -124,9 +131,8 @@ export const useMatrixHistogram = ({
             },
             error: (msg) => {
               if (!(msg instanceof AbortError)) {
-                notifications.toasts.addDanger({
+                notifications.toasts.addError(msg, {
```

Comment:
@patrykkopycinski @angorayc @XavierM Is this change here ok? `addDanger` only shows an error message, while `addError` allows the user to click to view full error details.